packages/espresso/example/ios/Flutter/AppFrameworkInfo.plist

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
```

Comment:
How do we generally handle the case that a plugin does not work for iOS? Espresso as a "plugin" won't work on iOS. We probably don't want to introduce any confusion that Espresso is going to work on iOS.

Reply:
Generally I think it's ok to check in "empty" platform implementations on unsupported platforms in order to get things to build, with an expectation that we'll remove them in the future. This is what we're doing for web plugins right now. I'll update the README to make it clear that this plugin is Android-only.